### DISCUSSION TITLE
About CloudStack installation issues

### DISCUSSION THREAD

**Discussion by Jonlli** [apache/cloudstack#9665] [Q&A]
hi,
i have a question . about vm and dedicated server issues.
1, vm rocky8.10  mysql 8.0
2. dedicated servers rocky8.10 mysql8.0
Both are installed in the same way.
the cloudstack install on the vm rocky8.10,it will work well. When it install dedicated servers rocky8.10. it will show 503 erro.
What happened?

**Comment by rajujith** [comment]
@Jonlli  can you cross-check the cloudstack-setup-databases command executed has the right database server IP? checking the management-server.log should give you more details on this issue.

**Comment by DaanHoogland** [comment]
closed this for lack of interaction, please reopen or open a new item if applicable
